index.bs

```diff
@@ -328,12 +341,25 @@ partial interface NeuralNetworkContext {
 ### mul ### {#api-neuralnetworkcontext-mul}
 <script type=idl>
 partial interface NeuralNetworkContext {
-  // Create an Operand object that represents the result of an element-wise
-  // binary multiplication operation with input operands a and b.
   Operand mul(Operand a, Operand b);
 };
 </script>
 
+<div algorithm=mul>
+    **Arguments:**
+        - *a*: an {{Operand}}. The first input tensor.
+        - *b*: an {{Operand}}. The second input tensor.
+
+    **Returns:** an {{Operand}}. The output tensor that contains the result of
+    element-wise binary multiplication of the two input tensors.
+
+    Computes the element-wise binary multiplication of the two input tensors.
+    The element-wise binary multiplication will be broadcasted accordingly by
```

Comment:
according to [[!numpy-broadcasting-rule]].

Reply:
Thanks @anssiko . Fixed it in new commit.